### PR TITLE
feat(api, package-operator): add multi chart packages

### PR DESCRIPTION
**Pull request by kosmoz**
## 📑 Description
Currently, a package can contain at most one helm release. For some packages (e.g. istio) this is not enough. This PR provides a backwards compatible solution that enables packages to contain multiple helm releases.

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

## ℹ Additional Information

Required for https://github.com/glasskube/packages/pull/75

### CODE REVIEW

File: config/crd/bases/packages.glasskube.dev_packageinfos.yaml [hunk 1]
```diff
@@ -191,6 +191,26 @@ spec:
                       chartVersion:
                         description: ChartVersion of the chart that should be installed.
                         type: string
+                      releases:
+                        items:
+                          properties:
+                            chartName:
```
Comment (pmig) [inline config/crd/bases/packages.glasskube.dev_packageinfos.yaml:197]: How is it possible to define different namespaces for these charts, as recommended by the istio installation?

Comment (kosmoz) [reply config/crd/bases/packages.glasskube.dev_packageinfos.yaml:197]: It's not possible. If you are talking about the istio ingress, it is an optional component and I would create a separate package for it.

Comment (pmig) [reply config/crd/bases/packages.glasskube.dev_packageinfos.yaml:197]: perfect

**Review by pmig — APPROVED** [review]

**Review by christophenne — APPROVED** [review]

**Comment by rgarcia** [comment]
curious if this feature is still planned? Looking to install istio which has multiple charts

**Comment by pmig** [comment]
> curious if this feature is still planned? Looking to install istio which has multiple charts

Hi @rgarcia we already created a first draft pr for the istio package (https://github.com/glasskube/packages/pull/75/files) already. Happy to continue working on it if it is relevant for you.

**Comment by rgarcia** [comment]
yes very relevant so I'd be thrilled if istio were available as a glasskube package! But I understand if it's too much trouble to update / merge this change to the operator

**Comment by pmig** [comment]
Gotcha, we will give you an update on the status tomorrow. :-)

**Comment by pmig** [comment]
Hi @rgarcia I just wanted to follow up if istio support and that PR is still relevant? We are ready to tackle it now :-)

**Comment by rgarcia** [comment]
Hey! In this particular case I was able to just use the istio-published "ambient" chart that pulls in the other charts: https://github.com/istio/istio/blob/master/manifests/sample-charts/ambient/Chart.yaml

and then made a glasskube package for it: https://github.com/onmetal-dev/metal/blob/main/glasskube/istio-ambient/v1.23.2%2B3/package.yaml feel free to bring it into the official packages repo

**Review by pmig — APPROVED** [review]
I tested the PR without regressions for already installed packages.

**Pull request merged**